code/modules/projectiles/guns/rifles.dm

```diff
@@ -1279,6 +1279,63 @@
 /obj/item/weapon/gun/rifle/l42a/training
 	current_mag = /obj/item/ammo_magazine/rifle/l42a/rubber
 
+//-------------------------------------------------------
+//-------------------------------------------------------
+
+//  M4RA Designated Marksmen Rifle
+
+/obj/item/weapon/gun/rifle/m4ra
+	name = "\improper M4RA Designated Marksman Rifle"
+	desc = "The M4RA is a designated marksman rifle fielded by the USCM. It was recently caught in a fierce competition with the L4 Battle Rifle, but ultimately the Colonial Marine Corps deemed the L4 too expensive to adopt and continued using the M4RA. Featuring bullpup ergonomics and utilising a higher velocity round than the M41, this is the perfect pick for any marine who prefers to keep a distance."
```

Comment:
```suggestion
	desc = "The M4RA is a designated marksman rifle fielded by the USCM. It was recently caught in a fierce competition with the L4 rifle series, but ultimately the Colonial Marine Corps deemed the L4 too expensive to adopt and continued using the M4RA. Featuring bullpup ergonomics and utilizing a higher velocity round than the M41, this is the perfect pick for any marine who prefers to keep a distance."
```
I've also talked with the concept artist who did some of the concept art stuff for ACM and ask where did the NSG23 (which is in the game) and the M4RA came from. For the case of the M4RA, it started out as the M42A, but since the game calls for underbarrel attachments (the shotgun and grenade launcher), GBX and Timegate (the other bois developing ACM) remodeled the M42A into the M4RA once the game releases. 

So take what information what you will.